contracts/profiles/StealthKeyResolver.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.7.4;
+import "../ResolverBase.sol";
+
+abstract contract StealthKeyResolver is ResolverBase {
+    bytes4 constant private STEALTH_KEY_INTERFACE_ID = 0x69a76591;
+
+    event StealthKeyChanged(bytes32 indexed node, uint256 generationPubKey, uint256 encryptionPubKey);
+
+    struct StealthKey {
+        uint256 generationPubKey;
+        uint256 encryptionPubKey;
+    }
+
+    mapping(bytes32=>StealthKey) _stealthKeys;
+
+    /**
+     * Sets the stealth keys associated with an ENS name, for anonymous sends.
+     * May only be called by the owner of that node in the ENS registry.
+     * @param node The node to update.
+     * @param generationPubKey The public key for generating a stealth address
+     * @param encryptionPubKey The public key for encrypting the generated address
+     */
+    function setStealthKeys(bytes32 node, uint256 generationPubKey, uint256 encryptionPubKey) external authorised(node) {
```

Comment:
As curve coordinates, they're integers, aren't they?

Reply:
They are! So either type should be fine. I don't think there would be a gas difference between the two, but if so that may be a good way to choose a type.

 I think I assumed the reason it'd be `bytes32` is for consistency, since the default `PublicResolver` [stores public key coordinates](https://docs.ens.domains/contract-api-reference/publicresolver#set-public-key) that way. So I guess my question for you is why was `bytes32` chosen for the public resolver?

Reply:
Good question. Seems like an oversight! Gas cost is the same either way.

Reply:
Fair enough, I have no objections to leaving this as-is then!